filters/encrypt/docs_test.go

```diff
@@ -62,6 +62,13 @@ func ExampleFilter() {
 		panic(err)
 	}
 
+	// ensure errors are returned if any of the nodes return an error while
+	// processing an event, since the event won't be written to the sink.  This
+	// is possible if you don't have a Filter that filters out events.
+	if err := b.SetSuccessThreshold(et, 1); err != nil {
+		// handle err
```

Comment:
Did you want this here or just the comment?
```
fmt.Println("err: ", err)
```

Reply:
ty so much for asking about this.  the question made me go back and validate how SetSuccessThreshold(...) works and this example was confusing and VERY incorrect.  I've removed the call to SetSuccessThreshold(...) in all the docs_test.go files.